puppet/zulip_ops/manifests/app_frontend.pp

```diff
@@ -4,6 +4,7 @@
   include zulip::rabbit
   include zulip::postfix_localmail
   include zulip::static_asset_compiler
+  include zulip::pip_requirements
```

Comment:
Let's do this in `app_frontend_base.pp`

Reply:
Deleted, because this is done in setup_Venv.py now.